source/files/stylesheets/adjustments.css

```diff
@@ -317,139 +311,11 @@ ul.doc-navigation {
   transition: color 0.2s linear;
 }
 
-.layout-footer .footer-row-1 {
-  margin: 0 auto;
-/*
-  max-width: 67em;
- */
-  padding: 0 10px;
-  min-height: 260px;
```

Comment:
Without this min-height, we lose the negative space between the bottom of the footer content and the trailing line of legal/copyright links.